circle.yml

```diff
@@ -82,10 +78,57 @@ jobs:
             make install_system_dependencies
             sudo ln -s /usr/lib/x86_64-linux-gnu/libmysqlclient.so /usr/lib/x86_64-linux-gnu/libmysqlclient.so.18
             bin/install-supported-firefox
-            bin/run-circleci-tests
-          no_output_timeout: 6h
+            mkdir -p /tmp/coverage
+            make << parameters.tests_type >>
+            if [ -e .coverage.* ]; then
+                cp -v .coverage.* /tmp/coverage/.
+            fi
+            if [ -d coverage ]; then
+                cp -vR coverage/* /tmp/coverage/.
+            fi
+          no_output_timeout: 2h
           environment:
             ANSIBLE_HOST_KEY_CHECKING: False
+      - persist_to_workspace:
+          root: /tmp
+          paths:
+              - coverage
+  coverage:
+    docker:
+      - image: circleci/python:3.5.4-node
```

Comment:
@viadanna, this and the following lines (and any other part not using Debian Stretch) should be updated to include the changes made in a733d982ddd7a6435f9b5771f55099ee4d142e28 to update the docker image and code to install the dependencies.

Reply:
Alright, I see only the build job had the updated image. I've updated the other ones and will push for another test.